pkg/server/server.go

```diff
@@ -306,6 +306,8 @@ func NewServerWithOpts(opts ...OpenFGAServiceV1Option) (*Server, error) {
 }
 
 func (s *Server) ListObjects(ctx context.Context, req *openfgav1.ListObjectsRequest) (*openfgav1.ListObjectsResponse, error) {
```

Comment:
What of `StreamedListObjects`? We should have the same metric observed in `StreamedListObjects` only under that RPC name instead of `ListObjects`.

Reply:
Added metrics for streamed list object